languages/rust/reference/exercise-concepts/macros.md

```diff
@@ -0,0 +1,31 @@
+# Concepts Required for the Macros exercise
+
+There are many ways to potentially solve an exercise, but I'll be looking at the [canonical example](https://github.com/exercism/rust/blob/666c366fc0d39d9f6fd8e837facea8afe2153ae2/exercises/macros/example.rs) as my primary reference for what concepts are used.
+
+When referring to the list of concepts, we mean [this list](https://github.com/exercism/v3/blob/4f164e3751202087ff40c8321238d4c283a90fd0/languages/rust/reference/README.md)
+
+Note that for both concept and practice exercises, the list of prerequisite concepts should be as small as possible. The goal here is not to enumerate all concepts which may be used in all possible approaches to the problem; it is to enumerate the minimal set of knowledge with which a determined student could conceivably solve the problem.
+
+Note that this exercise is very likely to become a concept exercise for the concept "Writing declarative macros".
+
+## Required concepts
+### Existing
+
+- structs
```

Comment:
should we add `macros` as  require concept to implementing declarative macros?
As someone who has developed mostly in high-level, loosely typed languages macros has been foreign to me.

Reply:
No: the whole point of this concept exercise is to teach how to implement declarative macros.